### ISSUE TITLE
Filings

### ISSUE THREAD

**Issue by LindsayYoung** [18F/openFEC#473]
Create a strategy of connecting data with links to the reports looking at the Data Warehouse.

**Comment by LindsayYoung** [comment]
I can only see one of the 2 tables that they gave me. I am following up with them now.

**Comment by LindsayYoung** [comment]
Ok, permissions added and looks like we got the data we wanted in `processed.f_rpt_or_form_sub_vw`

**Comment by LindsayYoung** [comment]
Modified the script to import the tables we want, now we will want to do a full refresh to get that data.

**Comment by rjmajma** [comment]
That's great, Lindsay. I'm going to close this issue. Can you please create a separate ticket in the backlog or put a note for yourself somewhere to make sure this is included in the data refresh. Given that we're waiting on some Golden Gate news, I think it's best if we hold off on refreshing till we know more.
